Source/Fuse.Controls.WebView/EvaluateJS.uno

```diff
@@ -105,20 +105,24 @@ namespace Fuse.Triggers.Actions
 	{
 		string _rawSource;
 		string _processedSource;
-		IWebView _target;
 
 		public event JSEventHandler Handler;
 
+		IWebView _target;
 		public IWebView WebView {
 			get { return _target; }
 			set { _target = value; }
 		}
-
+		
```

Comment:
There's no way to see these changes while editing. I also don't make them intentionally, they just happen as I move around the code.